frontend/src/employee-frontend/components/person-shared/PersonDetails.tsx

```diff
@@ -279,6 +279,18 @@ export default React.memo(function PersonDetails({
             )}
           </ButtonSpacer>
         ) : null}
+        {isChild &&
+        permittedActions.has('READ_ATTENDANCE_REPORT') &&
+        uiMode !== 'person-details-editing' ? (
+          <ButtonSpacer>
+            <Button
+              appearance="inline"
+              icon={faCalendar}
+              onClick={() => navigate(`/reports/child-attendance/${person.id}`)}
```

Comment:
Minusta tämän tulisi olla natiivi linkki nappulan sijaan. Tämä mahdollistaisi esim. uuteen tabiin avaamisen.

Reply:
Onks sille joku komponentti millä saa siitä kuitenkin tuon näköisen? Siinä vieressä on muita vastaavia niin ois hyvä olla visuaalisesti konsistentti kuitenkin.

Reply:
Ainakin näin on tehty `PersonDetails`:issa "Lataa osoitesivu" napin kohdalla:
```
           <a
              href={`${API_URL}/...`}
            >
              <Button
                appearance="inline"
                icon={faFileAlt}
                text={i18n.personProfile.downloadAddressPage}
                onClick={noop}
              />
            </a>
```